lib/inventory_refresh/persister.rb

```diff
@@ -143,6 +146,15 @@ def from_hash(persister_data, manager)
           persister.retry_max               = persister_data['retry_max']
           persister.total_parts             = persister_data['total_parts']
           persister.sweep_scope             = sweep_scope_from_hash(persister_data['sweep_scope'], persister.collections)
+          %i[
+            refresh_state_part_collected_at
+            refresh_state_part_sent_at
+            refresh_state_started_at
+            refresh_state_sent_at
+            ingress_api_sent_at
+          ].each do |timestamp_name|
+            persister.refresh_time_tracking[timestamp_name] = persister_data[timestamp_name.to_s]
+          end
```

Comment:
@slemrmartin is this going to work?

`from_hash(to_hash)` now if we're storing `refresh_time_tracking` in a hash but pulling it out of the top-level data?

Reply:
I didn't test it...probably `from_hash(to_hash)` don't work.
I'll fix it. 
@agrare Anyway do we use `persister.to_hash` somewhere? what is use case for this method?

Reply:
IMO just like `JSON.parse(a.to_json) == a` `InventoryCollection.from_hash(ic.to_hash) should == ic`